rllib/core/learner/torch/torch_differentiable_learner.py

```diff
@@ -273,6 +304,25 @@ def _convert_batch_type(
         batch = MultiAgentBatch(batch, env_steps=length)
         return batch
 
+    def _compute_off_policyness(self, batch):
```

Comment:
same: Let's try to re-use existing `TorchLearner` code, if possible.

Reply:
See my comment above.